management-web/src/main/resources/log4j2.yml

```diff
@@ -0,0 +1,28 @@
+Configutation:
+  name: Default
+  Properties:
+    Property:
+      name: log-path
+      value: "logs"
+  Appenders:
+    Console:
+      name: Console_Appender
+      target: SYSTEM_OUT
+      PatternLayout:
+        pattern: "%highlight{[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - Line %L - %msg%n}{FATAL=white, ERROR=red, WARN=blue, INFO=black, DEBUG=green, TRACE=blue}"
+    File:
+      name: File_Appender
+      fileName: ${log-path}/management-web.log
```

Comment:
What's the default log-path?

Reply:
log-path is a property inside the log4j2.yml with is a reference for a "logs" directory inside the project structure, or do you was expecting to log out on the local machine?

Reply:
it's fine, I just want to make sure you were aware of the location of the logs